app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -26,6 +26,11 @@ def textual_group_conditions
     h
   end
 
+  def textual_group_smart_management
+    items = %w(tags)
+    items.collect { |m| send("textual_#{m}") }.flatten.compact
+  end
```

Comment:
I think don't worry about it for now. I intend to do something for this general pattern in the near future, anyway.